core/src/main/java/nl/nn/adapterframework/doc/model/FrankElement.java

```diff
@@ -73,13 +75,18 @@ public FrankElement(final String fullName, final String simpleName, boolean isAb
 		this.allChildren = new HashMap<>();
 		this.allChildren.put(FrankAttribute.class, new LinkedHashMap<>());
 		this.allChildren.put(ConfigChild.class, new LinkedHashMap<>());
+		this.xmlElementNames = new ArrayList<>();
 	}
 
 	public void setParent(FrankElement parent) {
 		this.parent = parent;
 		this.statistics = new FrankElementStatistics(this);
 	}
```

Comment:
Ik weet niet precies wat er nu gebeurd maar ik zie in de JSON `parent: "java.lang.Object"` terugkomen. Ik denk dat het handig is dat parent alleen meegegeven wordt als deze ook een (Frank) parent heeft. En dus niet wanneer de parent een `java.lang.Object` is (wat altijd het geval is bij Java).

Reply:
We hebben er geen last van, want hij komt niet in de lijst van members van een groep. Het lijkt me zonde om extra code te maken alleen om Object weg te werken. Die code moet dan ook onderhouden worden.

Reply:
Was inderdaad een kleine moeite. Nu doen we er alleen relevante parents in. Gerrit, er het kan zijn dat er een paar corner cases zijn waarin we te veel parents tonen. Maar we zouden niet gaan scherpslijpen zoals we met de XSDs hebben gedaan.

Reply:
Super! Bedankt 😃